hw9/deco_file.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+
+
+def decorator(func):
+    def wrapper(*args, **kwargs):
+        t = datetime.now()
+        print(f'====\nFunction call time - {t.strftime("%H:%M:%S")}\n====')
+        func(*args, **kwargs)
+        with open('deco.txt', 'x') as deco_file:
```

Comment:
With the `x` modifier this decorator will be able to log only once. Change it to `a+`